client/src/api/cart/cartApi.js

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+import { BASE_URL } from "../baseUrl";
+
+export const getAllCartItemsApi = async () => {
+  const { data } = await axios.get(`${BASE_URL}/cart`, {

```

Comment:
When you'll make global API variable(attaching token in the req header), calling axios separately will be removed.

Reply:
resolved